draft-ietf-quic-load-balancers.md

```diff
@@ -813,6 +818,19 @@ to the correct forwrading address. Note that this solution is extensible to
 arbitrarily large numbers of load-balancing tiers, as the maximum server ID
 space is quite large.
 
+## Moving connections between servers
+
+Some deployments may transparently move a connection from one server to another.
+The means of transferring connection state between servers is out of scope of
+this document.
+
+To support a handover, a server involved in the transition could issue CIDs that
+map to the new server via a NEW_CONNECTION_ID frame, and retire CIDs associated
```

Comment:
Mention that the Retire Prior To field would be set to retire the previous CIDs.